requirements.txt

```diff
@@ -15,4 +15,4 @@ importlib_resources >= 5.0, < 5.1; python_version < '3.10'
 # NOTE: Ref: https://github.com/sarugaku/resolvelib/issues/69
 # NOTE: When updating the upper bound, also update the latest version used
 # NOTE: in the ansible-galaxy-collection test suite.
-resolvelib >= 0.5.3, < 0.10.0  # dependency resolver used by ansible-galaxy
+resolvelib >= 0.5.3, < 1.1.0  # dependency resolver used by ansible-galaxy
```

Comment:
To verify the new version works, temporarily change this to `resolvelib==1.0.1`.

Reply:
I'd rather extend https://github.com/ansible/ansible/blob/bad8843/test/integration/targets/ansible-galaxy-collection/vars/main.yml#L7-L11 with more versions.

Reply:
That should be updated as well.

@webknjaz Do you know if that covers all of our uses of resolvelib? If not, the temporary commit and revert would still be useful.

Reply:
also update for https://github.com/ansible/ansible/blob/bad8843/test/integration/targets/ansible-galaxy-collection/vars/main.yml#L7-L11

Reply:
@hswong3i Tests passed with the pinned version. Go ahead and revert that commit. No need to force push the revert, we'll just squash the commits when merging the PR.

Reply:
done ;-)